geth/account/accounts_test.go

```diff
@@ -100,12 +99,11 @@ func TestVerifyAccountPasswordWithAccountBeforeEIP55(t *testing.T) {
 	defer os.RemoveAll(keyStoreDir) //nolint: errcheck
 
 	// Import keys and make sure one was created before EIP55 introduction.
-	err = common.ImportTestAccount(keyStoreDir, "test-account3-before-eip55.pk")
-	require.NoError(t, err)
+	require.NoError(t, common.ImportTestAccount(keyStoreDir, "test-account4-public-pass.pk"))
```

Comment:
I'm not sure if this is correct, or if I need to generate a pre-eip55 keystore for this test.